swap.go

```diff
@@ -73,6 +73,12 @@ func GetHtlcScriptVersion(
 	return swap.HtlcV3
 }
 
+// IsTaproot returns true if the swap referenced by the passed swap contract
+// uses the v3 (taproot) htlc.
+func IsTaprootSwap(swapContract *loopdb.SwapContract) bool {
```

Comment:
nit: is it possible to make this a method on `swapContract`? imo it seems to be a bit cleaner.